src/server/router/server-config.js

```diff
@@ -0,0 +1,34 @@
+//server-config.js
+const express = require('express');
+const app = express();
+
+var cors = require('cors');
+var bodyParser = require('body-parser');
+
+//Question request-handler functions
```

Comment:
What about something with more explicit variable naming for these like: `postQuestion` etc. ?
The naming `handleGet` and similar is ambiguous and could be confusing as we add more handlers for things other than questions.